dev-libs/libprelude/files/libprelude-4.1.0-fix-python-bindings.patch

```diff
@@ -0,0 +1,38 @@
+--- a/bindings/python/Makefile.am	2017-07-17 12:03:05.653000000 +0200
```

Comment:
This looks identical to the old patch. Please don't introduce duplicate files, and just reuse the old patch.

(same for other packages)